sample/src/main/res/layout/rtl_fragment_view_pager_sample.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:indefinite_pager_indicator="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/description_text"
+        android:layout_width="wrap_content"
+        android:text="@string/rtl_viewpager_ref"
+        android:layout_centerHorizontal="true"
+        android:padding="8dp"
+        android:layout_height="wrap_content"/>
+
+    <com.rbrooks.indefinitepagerindicator.IndefinitePagerIndicator
+        android:id="@+id/viewpager_pager_indicator"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_margin="24dp"
+        android:layout_below="@+id/description_text"
+        indefinite_pager_indicator:dotColor="@color/colorPrimaryDark"
+        indefinite_pager_indicator:fadingDotCount="1"
+        indefinite_pager_indicator:selectedDotColor="@color/colorPrimary"
+        indefinite_pager_indicator:supportRTL="true"/>
+
+    <com.duolingo.open.rtlviewpager.RtlViewPager
+        android:id="@+id/viewpager"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_above="@+id/navigation_buttons_layout"
+        android:layout_below="@+id/viewpager_pager_indicator"/>
+
+    <RelativeLayout
+        android:id="@+id/navigation_buttons_layout"
+        style="?attr/buttonBarStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:background="@color/white"
+        android:elevation="4dp"
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/viewpager_previous_button"
+            style="@style/PrimaryFlatButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentLeft="true"
```

Comment:
Would you mind adding the ```layout_alignParentLeft="true"``` attribute to the ```viewpager_previous_button``` in ```fragment_view_pager_sample.xml```? The bottom nav bar doesn't render correctly in the normal ViewPager sample when in RTL layout.

Reply:
Sure thing! 🌮